ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-validator-ranGen.yaml

```diff
@@ -20,3 +20,4 @@ spec:
     - fileName: validatorCRs/informDuValidator.yaml
       remediationAction: inform
       policyName: "du-policy"
+      compliant: 5s
```

Comment:
This low setting is only valid if the validation policy is disconnected from the cluster at steady-state using a bindingExcludeRules entry with ztp-done. Please add a comment to help document this.